### DISCUSSION TITLE
(Live) Slider

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1963] [General]
The Carousel image rotator is very well done, and I'm wondering if that points the way forward for a content slider that can natively replace the existing Live Slider. Are there plans / ETA for a Slider block?

**Comment by meetmandeep** [comment]
What specific features are you looking for?

**Comment by eswanzey** [comment]
It seems that these days, the more elaborate animations are better handled by video for a number of reasons. And Block/Section animation lessens the need for a "Live Slider" 1:1 replacement. I think a case can be made for having the ability to animate designated image, text, or header blocks as foreground elements above a designated "background" element. The background element might be a Section or Grid containing whatever they contain.

Instead of a frame, slide, timeline or storyboard editor, all that would be needed is to be able to assign an existing animation type from the animations panel to each of these foreground elements. That, and something that can trigger the foreground animations to begin. For example, a Section animates into place as the page loads with a "Fade Up In" animation, and the completion of that animation would then need to trigger these foreground elements to then execute.

**Comment by meetmandeep** [comment]
As you said, the more elaborate animations are better handled as videos but sliders are great for SEO purposes. We'll keep this idea floating but we DO NOT intend to add timeline/storyboard etc. Presentation changes very quickly with new trends and becomes challenging to maintain.

**Comment by eswanzey** [comment]
Consider the ability to designate a trigger for an animation. If we say that the current default is that animations trigger from something like "page," add an ability to instead set the trigger to another block on the page with a simple text entry field. In this way, you provide the ability to cascade animations over and above the scrolling of the page being the driver. It would be a pretty cool feature I think.

**Comment by eswanzey** [comment]
A follow-on to my previous comment - some theme builders call it an off-canvas block. It opens up an elegant and sophisticated way to customize a page in a timeline fashion, the timeline being the scroll of the page. I think the concept behind that is very cool, that the scroll becomes the timeline that you can tap in on.

But specifically regarding animation triggers, I think there needs to be some control over what triggers an animation so that blocks can be more explicitly controlled. Probably some thing that's already discussed - an obvious next step on the core animation control that we have now.
